app/auth/forms.py

```diff
@@ -24,7 +24,7 @@ class RegistrationForm(FlaskForm):
                                                    Length(1, 64),
                                                    Regexp('^[A-Za-z]'
                                                    '[A-Za-z0-9_.]*$'), 0],
-                           'username must have only letters, numbers, '
+                           'username must have only letters, numbers, 
```

Comment:
Since we break the line to be in accordance to **pycodestyle**, the comma at the end is mandatory.